### PR TITLE
Log a unfinished.json file

### PR DESCRIPTION
**Pull request by ayan-b**
This will keep the dataypes and their corresponding
projects for which the ETL was not finished.

### CODE REVIEW

File: xena_gdc_etl/gdc2xena.py [hunk 1]
```diff
@@ -84,6 +86,12 @@ def gdc2xena(root_dir, projects, xena_dtypes):
             try:
                 dataset.download().transform().metadata()
             except Exception:
+                if project not in unfinished:
+                    unfinished[project] = [dtype]
+                else:
+                    unfinished[project].append(dtype)
+                with open("unfinished.json", "w") as outfile:
+                    json.dump(unfinished, outfile)
```
Comment (yunhailuo) [inline xena_gdc_etl/gdc2xena.py]: The file will be rewrite again every failed loop? And why not just print this summary at the end of log?

File: tests/test_gdc2xena.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import pytest
+
+from xena_gdc_etl import gdc2xena
+
+
+PATH = "tests/fixtures/"
+
+
+@pytest.mark.CI
+def test_logging():
```
Comment (yunhailuo) [inline tests/test_gdc2xena.py]: This is not real log if you don't put it to log. I'd suggest changing the name unless you do logging.

File: xena_gdc_etl/gdc2xena.py (hunk 1 above)
Comment (ayan-b) [reply xena_gdc_etl/gdc2xena.py]: This will allow us to check the unfinished datatypes on the fly.

Comment (ayan-b) [reply xena_gdc_etl/gdc2xena.py]: So, that we can run another session on the instance downloading those datatypes.

Comment (yunhailuo) [reply xena_gdc_etl/gdc2xena.py]: It's probably a bad idea for starting another session while the current session is still running and is already stressed server... But it's not a good solution anyway; so I'd say it's OK if this is good for you in real practice.

Another comment is where is this `unfinished.json`? Is it respecting the `root_dir` and other settings?

Comment (ayan-b) [reply xena_gdc_etl/gdc2xena.py]: Yup, now it does.

**Pull request merged**